Assets/Scripts/GyroRodInputHandler.cs

```diff
@@ -3,8 +3,8 @@
 public class GyroRodInputHandler : IRodInputHandler
 {
     private const float REACTION_DISTANCE = 1.6f; // ボールに反応する最大距離
-    private const float GYRO_SENSITIVITY = 1.0f; // ジャイロ感度
-    private const float ACCELERATION_SENSITIVITY = 1000f; // 加速度センサ感度
+    private float movementSensitivity = 1.0f; // 移動感度
+    private float rotationSensitivity = 300f; // 回転感度
```

Comment:
命名ルール統一のため、privateフィールドには変数の頭にアンダーバーを付けておこう。

```suggestion
    private float _movementSensitivity = 1.0f; // 移動感度
    private float _rotationSensitivity = 300f; // 回転感度
```